### ISSUE TITLE
Accessibility: MAS36: Keyboard: "Previous" button is not accessible with keyboard.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#42]
#A11yMAS
#MAS36
#Wipro
#vscode-website
#FireFox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in Edge.
2. Navigate to the "previous" buttons above "Meet IntelliSense." in middle of the page. [See screenshot]
3. Try to access the button.

**Actual Result:**
Unable to access the "previous" button with keyboard above "Meet IntelliSense" in middle of the page.

**Expected Result:**
The user should be able  to access the all the interactive elements available in the page.

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may find it difficult while navigating with the tab to use the Application. 

**Screenshots:**
![inspect view_previous button](https://cloud.githubusercontent.com/assets/24825441/21714305/47a5fb72-d424-11e6-8a7d-3258e61e9095.png)
![previous arrow button is not accessible with keyboard](https://cloud.githubusercontent.com/assets/24825441/21714304/47a5233c-d424-11e6-9f71-d4a274724e85.png)

**Comment by chrisdias** [comment]
I can access the Next and Previous buttons and pressing Enter when on those buttons scrolls through the social proof.
